pages/linux/imgp.md

```diff
@@ -0,0 +1,15 @@
+# imgp
+
+> Command line image resizer and rotator for JPEG and PNG images.
+
+- Convert single images and/or whole directories containing valid image formats:
```

Comment:
End the sentence with `... to a target resolution:`

Reply:
Thanks for keeping the high level for subbmision